docs/deployment.md

```diff
@@ -0,0 +1,41 @@
+# ESI Deployment
+
+ESI is composed of multiple OpenStack services. As a result, any OpenStack deployment tool that can deploy and configure the required services can create an ESI deployment. Our recommended deployment tool is [Standalone TripleO](https://docs.openstack.org/project-deploy-guide/tripleo-docs/latest/deployment/standalone.html). Standalone TripleO is part of the active upstream [TripleO project](https://docs.openstack.org/tripleo-docs/latest/), and is highly configurable.
+
+If you decide to use Standalone TripleO, you will need the following additional configurations before deployment.
+
+## Ironic Configuration
+
+Standalone TripleO requires the following configuration changes in order to deploy Ironic. Note that Ironic works better with openvswitch.
+
+In `standalone_parameters.yaml`, add the following to the `resource_registry` section:
+
+```
+  OS::TripleO::Services::NovaCompute: OS::Heat::None
```

Comment:
I would drop the indentation in your preformatted text here and elsewhere.

Reply:
Maybe show the `resource_registry` section header in this example.

Reply:
I'll add the headers and keep the indentation.